config/owasp/suppressions.xml

```diff
@@ -1,25 +1,16 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<suppressions	xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-
-  <!--Please add all the false positives under the below section-->
-  <suppress>
-    <notes>False Positives
-      CVE-2016-1000027  refer https://tools.hmcts.net/jira/browse/CCD-3249
-      CVE-2018-1258  refer https://tools.hmcts.net/jira/browse/CCD-460
-      CVE-2022-22978 refer https://tools.hmcts.net/jira/browse/CCD-3513
-      CVE-2021-22112 refer https://tools.hmcts.net/jira/browse/CCD-3514
-      CVE-2022-22976 refer https://tools.hmcts.net/jira/browse/CCD-3515
-    </notes>
-    <cve>CVE-2016-1000027</cve>
-    <cve>CVE-2018-1258</cve>
-    <cve>CVE-2022-22978</cve>
-    <cve>CVE-2021-22112</cve>
-    <cve>CVE-2022-22976</cve>
-  </suppress>
-  <!--End of false positives section -->
-
-  <!--Please add all the temporary suppression under the below section -->
-
-  <!--End of temporary suppression section -->
-
+<?xml version="1.0" encoding="UTF-8"?><suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+<suppress>
+<notes>False Positives
+        CVE-2016-1000027  refer https://tools.hmcts.net/jira/browse/CCD-3249
+        CVE-2018-1258  refer https://tools.hmcts.net/jira/browse/CCD-460
+        CVE-2022-22978 refer https://tools.hmcts.net/jira/browse/CCD-3513
+        CVE-2021-22112 refer https://tools.hmcts.net/jira/browse/CCD-3514
+        CVE-2022-22976 refer https://tools.hmcts.net/jira/browse/CCD-3515</notes>
+<cve>CVE-2018-1258</cve>
+<cve>CVE-2016-1000027</cve>
+<cve>CVE-2018-1258</cve>
```

Comment:
@marvincudjoe - I hope this new suppression was autogenerated. can you check why CVE-2018-1258 is repeated twice?

Reply:
Not autogenerated. Left by mistake by myself. Removed.